benchmarking/benchmark_definitions/resnet_cifar10.py

```diff
@@ -25,39 +25,28 @@
 )
 
 
-def resnet_cifar10_default_params(sagemaker_backend: bool):
+def resnet_cifar10_benchmark(sagemaker_backend: bool = False, **kwargs):
     if sagemaker_backend:
         instance_type = DEFAULT_GPU_INSTANCE_1GPU
     else:
         # For local backend, GPU cores serve different workers, so we
         # need more memory
         instance_type = DEFAULT_GPU_INSTANCE_4GPU
-    return {
-        "max_resource_level": 27,
-        "instance_type": instance_type,
-        "num_workers": 4,
-        "num_gpus": 1,
-        "dataset_path": "./",
-    }
-
-
-def resnet_cifar10_benchmark(sagemaker_backend: bool = False, **kwargs):
-    params = resnet_cifar10_default_params(sagemaker_backend)
     config_space = dict(
         _config_space,
-        **{MAX_RESOURCE_ATTR: params["max_resource_level"]},
-        dataset_path=params["dataset_path"],
-        num_gpus=params["num_gpus"],
+        **{MAX_RESOURCE_ATTR: 27},
+        dataset_path="./",
+        num_gpus=1,
     )
     _kwargs = dict(
-        script=Path(__file__).parent.parent.parent
+        script=Path(__file__).parent.parent
         / "training_scripts"
         / "resnet_cifar10"
         / "resnet_cifar10.py",
         config_space=config_space,
         max_wallclock_time=3 * 3600,
-        n_workers=params["num_workers"],
-        instance_type=params["instance_type"],
+        n_workers=4,
+        instance_type=instance_type,
```

Comment:
Can these continue to be specified through `params` or do they need to be hardcoded?

Reply:
These are just defaults, you can change them on the CL. The refactoring here is only to go from two functions to one, this is legacy stuff.